src/tools/skylark/java/com/google/devtools/skylark/skylint/DocstringUtils.java

```diff
@@ -361,9 +367,15 @@ DocstringInfo parse() {
       String summary = line;
       String nonStandardDeprecation = checkForNonStandardDeprecation(line);
       if (!nextLine()) {
+        if (summary.trim().isEmpty()) {
+          errors.add(new DocstringParseError("the docstring is empty", 1, ""));
+        }
         return new DocstringInfo(
             summary, Collections.emptyList(), "", nonStandardDeprecation, "", null);
       }
+      if (summary.trim().isEmpty()) {
+        errors.add(new DocstringParseError("the docstring is empty", 1, ""));
+      }
```

Comment:
So we're requiring function docstrings to be non-empty, but not rule/attr docstrings?

Reply:
i could add a check for empty string  rule/attr docstrings. what di you think?

Reply:
If we're going to prohibit empty docstrings then let's be consistent for other kinds of objects.

Can you use the error() helper instead of constructing DocstringParseError directly?